### DISCUSSION TITLE
Too slow in tmux

### DISCUSSION THREAD

**Discussion by herryao1997** [ohmysh/ohmysh#34] [Q&A]
I downloaded the oh my sh and configure the theme, but seems like every time when open tmux the ~/.bashrc is not sourced.
Then I added this code in the ~/.bash_profile:
```bash
if [ -f ~/.bashrc ] ; then
    source ~/.bashrc
fi
```
Then the problem is solved, but after that the loading time for the terminal is super long, I wonder is there any one facing similar issue as me? could you tell me how to fix this issue ?

thx very much about that!

**Comment by langonginc** [comment]
I don't know why it is so slow.

I'll fix `~/.bash_profile` problem.

**Comment by langonginc** [comment]
Maybe you can change to Konsole or other consoles?
